backend/data_tools/tests/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+
+services:
+
+  db:
+    image: "postgres:12"
+    platform: linux/amd64
+    container_name: unit-test-db
+    security_opt:
+      - no-new-privileges:true  # Resolve semgrep https://sg.run/0n8q
+    environment:
+      - POSTGRES_PASSWORD=local_password  # pragma: allowlist secret
+    read_only: true  # Resolve semgrep https://sg.run/e4JE
+    tmpfs: /var/run/postgresql/
+    ports:
+      - "54321:5432"
+    healthcheck:
```

Comment:
We technically don't need this here, if there's no other services in the stack that needs to wait on this healthcheck - especially since the pytest has a built-in "wait" function already. It doesn't hurt to keep it, just noting for reference.

Reply:
Removed.